browser/model/kompose.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+let path = require('path');
+
+import InstallableItem from './installable-item';
+import Installer from './helpers/installer';
+import Util from './helpers/util';
+import Platform from '../services/platform';
+
+class KomposeInstall extends InstallableItem {
+  constructor(installerDataSvc, downloadUrl, fileName, targetFolderName, sha256) {
+    super(KomposeInstall.KEY, downloadUrl, fileName, targetFolderName, installerDataSvc, false);
+    this.addOption('install', this.version, '', true);
+    this.sha256 = sha256;
+  }
+
+  static get KEY() {
+    return 'kompose';
+  }
+
+  installAfterRequirements(progress, success, failure) {
+    progress.setStatus('Installing');
+    let komposeDir = this.installerDataSvc.komposeDir();
+    let installer = new Installer(KomposeInstall.KEY, progress, success, failure);
+    let komposeExe = path.join(komposeDir, Platform.OS === 'win32' ? 'kompose.exe' : 'kompose');
+    installer.copyFile(
+      this.downloadedFile, komposeExe
+    ).then(()=> {
+      Platform.addToUserPath([this.option['install'].location]);
+      installer.succeed(true);
```

Comment:
Platform.addToUserPath returns a promise, so you should really wrap the next command into another .then clause.